File: learn_java/Begin/bai1.java
import java.util.Scanner;
public class bai1
{
    public static void main(String[] args)
    {
        double a,b;
        Scanner sc = new Scanner (System.in);
        System.out.println("input a: ");
        a= sc.nextDouble();
        System.out.println("input b: ");
        b= sc.nextDouble();

        System.out.println("abs = " + Math.abs(a));
        System.out.println("max = "+ Math.max(a,b));
        System.out.println("min = "+ Math.min(a,b));
        System.out.println("round = "+ Math.round(a));
        System.out.println("pow = "+ Math.pow(a,b));
    }
}

File: hoc_C_C++/bai1.cpp
#include <iostream>
#include <math.h>
using namespace std;
int gt (int n)
{
    int s=1;
    for (int i = 1; i <= n; i++)
    {
        s *=i;
    }
    return s;
}

int result (int n, int k)
{
    return gt(n)/(gt(k)*gt(n-k));
}
int main ()
{
    int n, k;
    cout<<"input n: ";
    cin>>n;
    cout<<"input k: ";
    cin>>k;
    int kq = result(n, k);
    cout<<"result "<<kq;
}